bats/schema-changes.bats

```diff
@@ -292,11 +281,11 @@ SQL
     dolt table rm test
     dolt sql <<SQL
 CREATE TABLE test (
-  c1 BIGINT COMMENT 'tag:1',
-  c2 BIGINT COMMENT 'tag:2',
-  c3 BIGINT COMMENT 'tag:3',
-  c4 BIGINT COMMENT 'tag:4',
-  c5 BIGINT COMMENT 'tag:5',
+  c1 BIGINT,
+  c2 BIGINT,
+  c3 BIGINT,
+  c4 BIGINT,
+  c5 BIGINT,
   PRIMARY KEY (c3)
 );
```

Comment:
@timsehn these were skipped in the last review. Removing the tag specification allows them to be dropped/added with new tags